src/lib/dev-environment/dev-environment-core.js

```diff
@@ -86,7 +87,9 @@ export async function createEnvironment( instanceData: NewInstanceData ) {
 		throw new Error( 'Environment already exists.' );
 	}
 
-	await prepareLandoEnv( instanceData, instancePath );
+	const cleanedInstanceData = cleanInstanceData( instanceData );
+
+	await prepareLandoEnv( instanceData, cleanedInstanceData );
```

Comment:
Shouldn't this be `prepareLandoEnv( cleanedInstanceData, instancePath )`?

Reply:
Great catch! thank you!